lib/service/connect.go

```diff
@@ -682,7 +682,7 @@ func (process *TeleportProcess) periodicSyncRotationState() error {
 	if _, err := process.WaitForEvent(process.GracefulExitContext(), TeleportReadyEvent); err != nil {
 		return nil
 	}
-	process.log.Infof("The new service has started successfully. Starting syncing rotation status with period %v.", process.Config.PollingPeriod)
+	process.log.Infof("The new service has started successfully. Starting syncing rotation status with period %v.", process.Config.Testing.PollingPeriod)
```

Comment:
It feels weird to put these timeouts/intervals under `Testing` and then just use them in product code. They feel like a legitimate configuration option that we just haven't exposed yet. This is the actual polling period, it's not like there is some other non-testing polling period. Just my 2c doesn't really make a difference

Reply:
Yep, but they only exist so tests could override them, otherwise it would be just default values 🤷 
But I agree it's weird, as I wrote in the description. I pulled `PollingPeriod` back to the main config, as the most heavily used "almost as normal" and so the most weird one.